doc/descriptors.md

```diff
@@ -10,6 +10,13 @@ Supporting RPCs are:
 - `deriveaddresses` takes as input a descriptor and computes the corresponding
   addresses.
 - `listunspent` outputs a specialized descriptor for the reported unspent outputs.
+- `getaddressinfo` - outputs a descriptor for solvable addresses (since v0.18).
+- `importmulti` - takes as input descriptors to import into the wallet
+  (since v0.18).
+- `generatetodescriptor` - takes as input a descriptor and generates coins to it
+  (`regtest` only, since v0.19).
+- `utxoupdatepsbt` - takes as input descriptors to add information to the psbt
```

Comment:
Apologies for not seeing it before, but the new entries add a hyphen (" - ") between the rpc name and the description which the existing entries do not have.

Reply:
Huh wonder why I thought there were hyphens... anyways fixed.